### ISSUE TITLE
tput_test in the CI does nothing?

### ISSUE THREAD

**Issue by valassi** [madgraph5/madgraph4gpu#937]
This is related to PR #918 to fix the couplings issue #826

The patch modifies the code of susy_gg_t1t1, and alows the CI tmad_test to succeed fixing #826 

However, the tput test which include runTest.exe should normally fail unless the reference log is regenerated. In PR #918 the CI test succeeds instead.

I had a look
https://github.com/madgraph5/madgraph4gpu/actions/runs/9980359084/job/27581744124?pr=918
```

*******************************************************************************
*** tput_test susy_gg_t1t1.mad (P1_gg_t1t1x)
*******************************************************************************

Testing in /home/runner/work/madgraph4gpu/madgraph4gpu/epochX/cudacpp/susy_gg_t1t1.mad/SubProcesses/P1_gg_t1t1x

(SKIP cuda because nvcc is missing on this node)

(SKIP cpp512y which is not supported - no avx512vl in /proc/cpuinfo)

(SKIP cpp512z which is not supported - no avx512vl in /proc/cpuinfo)

@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
[testsuite_oneprocess.sh] tput_test (susy_gg_t1t1.mad) finished with status=0 (OK) at Wed Jul 17 19:47:14 UTC 2024
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
```

My impression is that this did strictly nothing? Or I might be wrong? In any case, the test should at least made a bit more verbose.

**Comment by valassi** [comment]
Investigating in #938. Here is a bug
https://github.com/madgraph5/madgraph4gpu/actions/runs/10061775347/job/27812750482?pr=938
```

@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
[testsuite_oneprocess.sh] tput_test (gg_tt.mad) starting at Tue Jul 23 15:28:43 UTC 2024
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
Current directory is /home/runner/work/madgraph4gpu/madgraph4gpu/epochX/cudacpp/gg_tt.mad
FPTYPE=d

*******************************************************************************
*** tput_test gg_tt.mad (P1_gg_ttx)
*******************************************************************************

Testing in /home/runner/work/madgraph4gpu/madgraph4gpu/epochX/cudacpp/gg_tt.mad/SubProcesses/P1_gg_ttx
-------------------------------------------------------------------------------
--- tput_test gg_tt.mad (P1_gg_ttx) BACKEND=backend
-------------------------------------------------------------------------------


(SKIP cuda because nvcc is missing on this node)
-------------------------------------------------------------------------------
--- tput_test gg_tt.mad (P1_gg_ttx) BACKEND=backend
-------------------------------------------------------------------------------

ls: cannot access 'build.hip*': No such file or directory
DEBUG: 
-------------------------------------------------------------------------------
--- tput_test gg_tt.mad (P1_gg_ttx) BACKEND=backend
-------------------------------------------------------------------------------
ls: cannot access 'build.cppnone*': No such file or directory

DEBUG: 
-------------------------------------------------------------------------------
--- tput_test gg_tt.mad (P1_gg_ttx) BACKEND=backend
-------------------------------------------------------------------------------

ls: cannot access 'build.cppsse4*': No such file or directory
DEBUG: 
-------------------------------------------------------------------------------
--- tput_test gg_tt.mad (P1_gg_ttx) BACKEND=backend
-------------------------------------------------------------------------------

ls: cannot access 'build.cppavx2*': No such file or directory
DEBUG: 
-------------------------------------------------------------------------------
--- tput_test gg_tt.mad (P1_gg_ttx) BACKEND=backend
-------------------------------------------------------------------------------


(SKIP cpp512y which is not supported - no avx512vl in /proc/cpuinfo)
-------------------------------------------------------------------------------
--- tput_test gg_tt.mad (P1_gg_ttx) BACKEND=backend
-------------------------------------------------------------------------------


(SKIP cpp512z which is not supported - no avx512vl in /proc/cpuinfo)

@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
[testsuite_oneprocess.sh] tput_test (gg_tt.mad) finished with status=0 (OK) at Tue Jul 23 15:28:43 UTC 2024
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
```

Will fix this tomorrow

**Comment by valassi** [comment]
This is fixed by #938 - closing
